pack/side/wog_encounter.json

```diff
@@ -0,0 +1,1035 @@
+        "illustrator": "Ethan Patrick Harris",
+        "name": "Cloak of the Outer Realm",
+        "pack_code": "wog",
+        "position": 51,
+        "quantity": 1,
+        "skill_agility": 2,
+        "skill_wild": 1,
+        "slot": "Body",
+        "subname": "Interwoven Distortion",
+        "text": "Uses (4 charges).\n[reaction] When a non-[[Elite]] enemy would engage you, spend 1 charge: Exhaust that enemy instead.",
+        "traits": "Item. Relic. Clothing.",
+        "type_code": "asset"
+    },
+    {
+        "code": "86052",
+        "cost": 2,
+        "deck_limit": 1,
+        "encounter_code": "war_of_the_outer_gods",
+        "encounter_position": 48,
+        "faction_code": "neutral",
+        "illustrator": "Quintin Gleim",
+        "name": "Pocket Portal",
+        "pack_code": "wog",
+        "position": 52,
+        "quantity": 1,
+        "skill_wild": 2,
+        "subname": "Doorway to Another World",
+        "text": "Fast.\n<b>Forced</b> - After Pocket Portal enters play: Place your investigator mini-card onto Pocket Portal and immediately end your turn. While your mini-card is on Pocket Portal, you are not at any location and enemies cannot spawn engaged with you. At the start of your turn, discard Pocket Portal and move to any revealed location.",
+        "traits": "Spell.",
+        "type_code": "asset"
+    },
+    {
+        "code": "86053",
+        "cost": 1,
+        "deck_limit": 1,
+        "encounter_code": "war_of_the_outer_gods",
+        "encounter_position": 49,
+        "faction_code": "neutral",
+        "illustrator": "Robert Laskey",
+        "name": "Dreaded End",
+        "pack_code": "wog",
+        "position": 53,
+        "quantity": 1,
+        "skill_wild": 1,
+        "skill_willpower": 2,
+        "slot": "Arcane",
+        "subname": "Gift of the Void",
+        "text": "Uses (3 charges).\n[action] Spend 1 charge: Choose one -\n- Place a non-[[Elite]] enemy at your location on top of the encounter deck.\n- Look at the top 5 cards of the encounter deck. Discard 2 of them and return the rest to the top, in any order.",
+        "traits": "Spell.",
+        "type_code": "asset"
+    },
+    {
+        "code": "86054",
+        "cost": 1,
+        "deck_limit": 1,
+        "encounter_code": "war_of_the_outer_gods",
+        "encounter_position": 50,
+        "faction_code": "neutral",
+        "illustrator": "Robert Laskey",
+        "name": "Blade of Ark'at",
+        "pack_code": "wog",
+        "position": 54,
+        "quantity": 1,
+        "skill_combat": 2,
+        "skill_wild": 1,
+        "slot": "Hand",
+        "subname": "Shellblade Tribute",
+        "text": "[free] Exhaust Blade of Ark'at and spend 1 resource: Place 1 resource on Blade of Ark'at.\n[action]: <b>Fight</b>. You get +1 [combat] for each resource on Blade of Ark'at. If you succeed, remove up to 3 resources from Blade of Ark'at to deal that much additional damage to the attacked enemy.",
```

Comment:
For the first ability, please add in italics '(from the token pool)' after Blade of Ark'at: 
Place 1 resource on Blade of Ark'at <i>(from the token pool)</i>.

Reply:
For Fight, please move the full stop inside the bold tag: <b>Fight.</b>